vizro-core/examples/visual-vocabulary/README.md

```diff
@@ -95,7 +95,7 @@ The dashboard is still in development. Below is an overview of the chart types f
 | Treemap               | ✅     | Part-to-whole            | [Treemap with px](https://plotly.com/python/treemaps/)                                     | [px.treemap](https://plotly.com/python-api-reference/generated/plotly.express.treemap.html)                                                                                                     |
 | Venn                  | ❌     | Part-to-whole            |                                                                                            |                                                                                                                                                                                                 |
 | Violin                | ✅     | Distribution             | [Violin plot with px](https://plotly.com/python/violin/)                                   | [px.violin](https://plotly.com/python-api-reference/generated/plotly.express.violin.html)                                                                                                       |
-| Waterfall             | ❌     | Part-to-whole, Flow      |                                                                                            |                                                                                                                                                                                                 |
+| Waterfall             | ✅     | Part-to-whole, Flow      | [Waterfall charts in Python](https://plotly.com/python/waterfall-charts/)                  |                                                                                                                                                                                                 | [go.Waterfall](https://plotly.github.io/plotly.py-docs/generated/plotly.graph_objects.Waterfall.html) |
```

Comment:
To stay consistent with the phrasing of charts that use `go` charts and to give credits to Plotly :)

```suggestion
| Waterfall             | ✅     | Part-to-whole, Flow      | [Waterfall charts in Plotly](https://plotly.com/python/waterfall-charts/)                  |                                                                                                                                                                                                 | [go.Waterfall](https://plotly.github.io/plotly.py-docs/generated/plotly.graph_objects.Waterfall.html) |
```